### ISSUE TITLE
Trailing dot

### ISSUE THREAD

**Issue by RazrFalcon** [Alexhuszagh/rust-lexical#4]
```rust
println!("{}", 1.0.to_string()); // 1

let mut buf = [0u8, 64];
lexical_core::ftoa::f64toa_slice(v, 10, &mut buf);
println!("{}", std::str::from_utf8(buf).unwrap()); // 1.
```

Is this an intended behavior?

**Comment by RazrFalcon** [comment]
`slc` in your first example will also return `1.`

The actual output isn't that important for me. I just thought that it should much the Rust one.

PS:  `v` is `1.0` in my example. Forget to fix it.

**Comment by RazrFalcon** [comment]
Whoops. By bad. The correct syntax for arrays is `[0u8; 64]`, not `[0u8, 64]`. Missed this one. 

Maybe it should panic or something when the provided buffer is too small?

**Comment by Alexhuszagh** [comment]
@RazrFalcon That might be a good idea, however, that might be considered a feature by some. I currently use a temporary buffer [internally](https://github.com/Alexhuszagh/rust-lexical/blob/master/lexical-core/src/ftoa/api.rs#L137), and then just copy the first `n` bytes into the buffer when the buffer is too small. You may be right that this is an anti-feature, rather than a feature, however.

**Comment by Alexhuszagh** [comment]
This also means that handling "-0" is somewhat odd with a trimmed float. Both "+0.0" and "-0.0" are different in IEEE754 floats, since the sign bit is different, however, "+0" and "-0" do not make sense as integers.

I currently export "-0.0" for the float designed by "-0.0", while Rust exports "0". I think that the sign is a feature, since the data type is a float, and that data is required for an accurate representation of the binary float. Thoughts?

**Comment by RazrFalcon** [comment]
I think that silent/ignored errors is anti-pattern in Rust.

As for `-0` - I'm fine with the current Rust behavior.

**Comment by Alexhuszagh** [comment]
This is featured-gated as of 492ec82ee66dbcbab10b9fedfd16b90a8ec08dea with the "trim_floats" feature. It also panics for both integers and floats if the provided buffer size is smaller than `MAX_XX_SIZE`, where `XX` is something in the set {I8, U8, F32, ...}.

The variables for these buffer sizes have also been publicly exposed from the config, and therefore should appear in a soon-to-be-published minor release. I'll keep the issue open until I push that fix.

**Comment by Alexhuszagh** [comment]
Fixed as of lexical-core 0.2.
